src/backend/gporca/libgpopt/src/translate/CTranslatorExprToDXL.cpp

```diff
@@ -7553,10 +7553,15 @@ CTranslatorExprToDXL::MakeDXLTableDescr(const CTableDescriptor *ptabdesc,
 
 	CMDIdGPDB *mdid = CMDIdGPDB::CastMdid(ptabdesc->MDId());
 	mdid->AddRef();
-
+                                                                                                                                                                                                                       
 	CDXLTableDescr *table_descr = GPOS_NEW(m_mp)
 		CDXLTableDescr(m_mp, mdid, pmdnameTbl, ptabdesc->GetExecuteAsUserId());
 
+	if (ptabdesc->IsAliasUsed()) {
```

Comment:
```suggestion
	if (NULL != ptabdesc->Alias()) {
```

Reply:
This won't work, CTableDescriptor::Alias() returns reference, not a pointer. Basically, this is a reason for introducing CTableDescriptor::IsAliasUsed().